warehouse/packaging/models.py

```diff
@@ -622,6 +622,16 @@ def __table_args__(cls):  # noqa
     uploader: Mapped[User] = orm.relationship(User)
     uploaded_via: Mapped[str | None]
 
+    @property
+    def name(self) -> str:
+        """
+        Release doesn't have a name column, inherit from its Project.
+
+        NOTE: `name` is used in `warehouse.packaging.tasks.sync_bigquery_release_files`
+        If changed here and not there, may have implications on downstream data.
+        """
+        return self.project.name
```

Comment:
should this (or the property suggested above) perhaps be `f”{self.project.name}@{self.version}`? or similar to not discard too much fidelity?

Reply:
Dropped this scope, and using `repr()` instead.